applications/RANSApplication/tests/cpp/test_incompressible_potential_flow_conditions.cpp

```diff
@@ -0,0 +1,282 @@
+    // setting reference values
+    ref_RHS = ZeroVector(2);
+    ref_LHS = ZeroMatrix(2, 2);
+
+    KRATOS_CHECK_VECTOR_NEAR(RHS, ref_RHS, 1e-12);
+    KRATOS_CHECK_MATRIX_NEAR(LHS, ref_LHS, 1e-12);
+
+    // checking for inlet condition
+    r_condition.SetValue(RANS_IS_INLET, 1);
+    r_condition.Initialize();
+    r_condition.CalculateLocalSystem(
+        LHS, RHS, static_cast<const ProcessInfo>(r_model_part.GetProcessInfo()));
+    // settting reference values
+    ref_RHS[0] = -7.4912949377611748e-01;
+    ref_RHS[1] = -7.4912949377611748e-01;
+
+    KRATOS_CHECK_VECTOR_NEAR(RHS, ref_RHS, 1e-12);
+    KRATOS_CHECK_MATRIX_NEAR(LHS, ref_LHS, 1e-12);
+}
+
+KRATOS_TEST_CASE_IN_SUITE(RansIncompressiblePotentialFlowVelocityInlet2D2N_CalculateLeftHandSide,
+                          KratosRansFastSuite)
+{
+    // Setup:
+    Model model;
+    auto& r_model_part = RansIncompressiblePotentialFlowVelocityInlet2D2N_SetUp(model);
+
+    // Test:
+    Matrix LHS, ref_LHS(2, 2, 0.0);
+    auto& r_condition = r_model_part.Conditions().front();
+
+    r_condition.SetValue(RANS_IS_INLET, 0);
+    r_condition.Initialize();
+    r_condition.CalculateLeftHandSide(
+        LHS, static_cast<const ProcessInfo>(r_model_part.GetProcessInfo()));
+
+    KRATOS_CHECK_MATRIX_NEAR(LHS, ref_LHS, 1e-12);
+
+    r_condition.SetValue(RANS_IS_INLET, 1);
+    r_condition.Initialize();
+    r_condition.CalculateLeftHandSide(
+        LHS, static_cast<const ProcessInfo>(r_model_part.GetProcessInfo()));
+
+    KRATOS_CHECK_MATRIX_NEAR(LHS, ref_LHS, 1e-12);
+}
+
+KRATOS_TEST_CASE_IN_SUITE(RansIncompressiblePotentialFlowVelocityInlet2D2N_CalculateRightHandSide,
+                          KratosRansFastSuite)
+{
+    // Setup:
+    Model model;
+    auto& r_model_part = RansIncompressiblePotentialFlowVelocityInlet2D2N_SetUp(model);
+
+    // Test:
+    Vector RHS, ref_RHS(2, 0.0);
+    auto& r_condition = r_model_part.Conditions().front();
+
+    r_condition.SetValue(RANS_IS_INLET, 0);
+    r_condition.Initialize();
+    r_condition.CalculateRightHandSide(
+        RHS, static_cast<const ProcessInfo>(r_model_part.GetProcessInfo()));
+
+    r_condition.SetValue(RANS_IS_INLET, 1);
+    r_condition.Initialize();
+    r_condition.CalculateRightHandSide(
+        RHS, static_cast<const ProcessInfo>(r_model_part.GetProcessInfo()));
+    ref_RHS[0] = -7.4912949377611748e-01;
+    ref_RHS[1] = -7.4912949377611748e-01;
+
+    KRATOS_CHECK_VECTOR_NEAR(RHS, ref_RHS, 1e-12);
+}
+
+KRATOS_TEST_CASE_IN_SUITE(RansIncompressiblePotentialFlowPressureBodyForce2D2N_EquationIdVector,
+                          KratosRansFastSuite)
+{
+    // Setup:
+    Model model;
+    auto& r_model_part = RansIncompressiblePotentialFlowPressureBodyForce2D2N_SetUp(model);
+
+    // Test:
+    RansApplicationTestUtilities::TestEquationIdVector<ModelPart::ConditionsContainerType>(r_model_part);
+}
+
+KRATOS_TEST_CASE_IN_SUITE(RansIncompressiblePotentialFlowPressureBodyForce2D2N_GetDofList,
```

Comment:
```suggestion
KRATOS_TEST_CASE_IN_SUITE(RansIncompressiblePotentialFlowPressureBodyForce2D2NGetDofList,
```

Reply:
This I would like to keep as it is because its a long name, having underscores gives little bit of readability :)